spec/supabase_kt_v1.yml

```diff
@@ -37,8 +37,8 @@ functions:
       **On JS**, it uses the website origin as the callback url. Session importing gets handled automatically.
       **Windows, tvOS, watchOS & Linux** currently have no default implementation. Feel free to create a PR.
 
-      You always make your own implementation and use `gotrue.parseSessionFromFragment(fragment)` or `gotrue.parseSessionFromUrl(url)` to let [supabase-kt](https://github.com/supabase-community/supabase-kt) handle the parsing after receiving a callback.
-      Then you can simply use `gotrue.importSession(session)`.
```

Comment:
@jan-tennert 
I think it would make sense to leave this `supabase_kt_v1.yml` file and create a new `supabase_kt_v2.yml` file for the new v2 docs like how the JS SDK does it, but what do you think?

Reply:
Yea sounds like a good idea